packages/core/src/webapp/util/ascii-to-table.ts

```diff
@@ -383,7 +463,7 @@ export const formatTable = (entityType: string, lines: Pair[][]): Table => {
       name: nameForDisplay,
       nameDom: columns[0].valueDom,
       fontawesome: idx !== 0 && columns[0].key === 'CURRENT' && 'fas fa-network-wired',
-      onclick: false,
+      onclick: isKube && nameColumnIdx === 0 && onclick, // if the first column isn't the NAME column, no onclick; see onclick below
```

Comment:
Added isKube to onclick to white-list oc. All other changes are from reverting commit 39500ba .